TCAT/WhatsNewHeaderView.swift

```diff
@@ -0,0 +1,155 @@
+//
+//  WhatsNewPopupView.swift
+//  TCAT
+//
+//  Created by Omar Rasheed on 9/26/18.
+//  Copyright © 2018 cuappdev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+protocol WhatsNewDelegate {
+    func okButtonPressed()
+    func cardPressed()
+}
+
+class WhatsNewHeaderView: UIView {
+
+    var whatsNewDelegate: WhatsNewDelegate?
+    var updateTitle: UILabel!
+    var dismissButton: UIButton!
+    var updateDescription: UILabel!
+    var whatsNewHeader: UILabel!
+    var backgroundView: UIView!
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        createBackgroundView()
+        createUpdateDescription()
+        createWhatsNewHeader()
+        createUpdateTitle()
+        createDismissButton()
+        updateConstraints()
+    }
+    
+    func createBackgroundView() {
+        backgroundView = UIView()
+        backgroundView.backgroundColor = .white
+        backgroundView.layer.cornerRadius = 14
+        backgroundView.clipsToBounds = true
+        backgroundView.layer.borderWidth = 1
+        backgroundView.layer.borderColor = UIColor(red: 233/255, green: 235/255, blue: 238/255, alpha: 1).cgColor
```

Comment:
Use `UIColor.tableBackgroundColor` for now. We'll better organize these colors with design in a future refactor, but that's the closest one to what is in Zeplin.